bin/push_docker.js

```diff
@@ -32,16 +31,15 @@ try {
   currentBranch = execSync('git rev-parse HEAD').toString().trim();
 } catch (error) {}
 
-const distributionParam = process.argv[2];
-const stageParam = process.argv[3];
-const suffix = distributionParam ? `-${distributionParam}` : '';
-const stage = stageParam ? `-${stageParam}` : '';
+const distributionParam = process.argv[2] ? process.argv[2] : '';
+const stageParam = process.argv[3] ? process.argv[3] : '';
+const releaseParam = process.argv[4] ? process.argv[4] : '';
 const commitSha = process.env.GITHUB_SHA || 'COMMIT_ID';
 const commitShaLength = 7;
 const commitShortSha = commitSha.substring(0, commitShaLength - 1);
-const configurationEntry = suffix
-  ? `wire-web-config-default${suffix}`
-  : `wire-web-config-default${currentBranch === 'master' ? '-master' : '-staging'}`;
+const configurationEntry = distributionParam
+  ? `wire-web-config-default${distributionParam ? `-${distributionParam}` : ''}`
```

Comment:
```suggestion
  ? `wire-web-config-default-${distributionParam}`
```

Reply:
When `distributionParam` is an empty string the result would be `wire-web-config-default-`.

Reply:
But then this string wouldn't even be used. We are already checking if `distributionParam` is truthy in the line above.

Reply:
Ah right, got confused by the GH quick view.

Reply:
My own refactoring caught up wit me. thx!